Modules/Devices/src/Devices.Infrastructure/Persistence/Database/ValueConverters/TierIdEntityFrameworkValueConverter.cs

```diff
@@ -19,13 +19,13 @@ public TierIdEntityFrameworkValueConverter(ConverterMappingHints? mappingHints)
     }
 }
 
-public class NullableTierIdValueConverter : ValueConverter<TierId?, string?>
```

Comment:
Keep the `?`s please. Same for the other `Nullable*ValueConverters`.
This will be necessary once we activate nullable reference types.